docs/economy/introduction.md

```diff
@@ -27,16 +27,21 @@ First, there’s the game.
 
 Already we have an emergent multiplayer game with interesting psychological dynamics. Do I wait to bet big? Do I load up my gladiators with the maximum allowed items and then wait? Do I bet big and *then* come in and fill up items on other gladiators. Each tournament is like a giant game of bridge but instead of cards we have gory battles and AI-enhanced pre-battle taunting.
 
-Everything in the game is a token. PTG is a token, the gladiators are NFTs, the items are NFTs, the Factions themselves are tokens which can be owned. Owners of factions may transfer PTG out of factions should they desire and owners of gladiator NFTs may also transfer the gladiator’s PTG.
+![image](https://siasky.net/3AGwFmhT1mFpKdGauqYT2D9qozXG-XBf54F5W6z5WUABXg)
 
-Betting lets players stack PTG and they only need a little luck (and a good ITEM strategy). Owning gladiator contracts or factions (or parts thereof) earns you passive PTG.
+Everything in the game is a token. PTG is a token, the gladiators are NFTs, the items are NFTs, the Factions themselves are tokens which can be owned. Owners of factions may transfer PTG out of factions should they desire and owners of gladiator NFTs may also transfer out the gladiator’s PTG.
+
+Betting lets players stack PTG and they only need a little luck (and a good ITEM strategy). Bonuses added to the prize pools will be larger earlier on and be reduced over time.  This gives the advantage to those that get in early.
+
+Owning gladiator contracts or factions (or parts thereof) earns you passive PTG.  This is primarily done through staking PTG into a faction.  PTG can then be farmed into buying bigger shares of existing factions, retaining the services of your own gladiator or even starting a new gladiator or faction and working them up through the laddering system.
```

Comment:
Oh I didn't see this before - this doesn't make sense. The staking should be about putting USDC/PTG LP tokens into something, but we don't have that figured out yet. Also the "farming" bit doesn't make a lot of sense here ither. So maybe we just drop this paragraph?

Reply:
wait you marked this as resolved but it's still in there?